formatTest/typeCheckedTests/input/reasonLineComments.rei

```diff
@@ -0,0 +1,3 @@
+module JustString : {
+  include Map.S; // Comment eol include
```

Comment:
We will need to get the syntax highlighters to be updated. Vim should be good. But imagine if you had a brace or mismatched paren in a comment.